### PR TITLE
Made an avatar component with default avatar image

### PR DESCRIPTION
**Pull request by DELTAmats**
added it to the page component

### CODE REVIEW

File: components/page-component/PageComponent.vue [hunk 1]
```diff
@@ -8,8 +8,11 @@
         </li>
       </ul>
       <div class="storyteller" v-if="page.storyteller">
-        {{page.storyteller.name}}
-        {{page.storyteller.avatar}}
+        <avatar-component
+          v-bind:avatar="page.storyteller.avatar"
+          v-bind:name="page.storyteller.name"
+          />
+          {{page.storyteller.name}}
```
Comment (jurgenbelien) [inline components/page-component/PageComponent.vue]: I'd move the name inside the component

File: components/page-component/PageComponent.vue [hunk 2]
```diff
@@ -37,6 +40,7 @@
 
 <script>
 import '~/components/colors/colors.css'
+import AvatarComponent from '~/components/avatar-component/avatarComponent'
```
Comment (jurgenbelien) [inline components/page-component/PageComponent.vue]: AvatarComponent filename should start with a capital

File: components/avatar-component/README.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Avatar
+
+Returns an image, default image if no src is supplied
+otherwise an image from supplied avatar data path and imgixHost
+storyteller name is for the title attribute
+
+## How to use
+
+`<avatar-component v-bind:avatar="storyteller" v-bind:avatar="storytellername" />`
```
Comment (jurgenbelien) [inline components/avatar-component/README.md]: This does not correspond with the actual template

File: components/avatar-component/AvatarComponent.vue [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  </div>
+  </template>
+
+<script>
+import '~/components/colors/colors.css'
+import _defaultAvatarSrc from './assets/Portrait_Placeholder.png'
+
+export default {
+  props: {
+    avatar: {
```
Comment (jurgenbelien) [inline components/avatar-component/AvatarComponent.vue]: I'd call this `image`

File: components/avatar-component/AvatarComponent.vue [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+      type: Object
+    },
+    name: {
+      type: String
+    }
+    // 'storytellerAvatar',
+    // 'storyteller',
+  },
+  computed: {
+    avatarSrc: function () {
```
Comment (jurgenbelien) [inline components/avatar-component/AvatarComponent.vue]: you can make this shorthand `avatarSrc () {` (or `imgSrc () {` if you had changed the aforementioned name

File: components/avatar-component/README.md (hunk 3 above)
Comment (DELTAmats) [reply components/avatar-component/README.md]: changed to <avatar-component
  v-bind:avatar="page.storyteller.avatar"
  v-bind:name="page.storyteller.name"
  />

Comment (DELTAmats) [reply components/avatar-component/README.md]: <avatar-component
  v-bind:avatar="page.storyteller.avatar"
  v-bind:name="page.storyteller.name"
  />

File: components/avatar-component/AvatarComponent.vue (hunk 4 above)
Comment (DELTAmats) [reply components/avatar-component/AvatarComponent.vue]: changed to img

File: components/avatar-component/AvatarComponent.vue (hunk 5 above)
Comment (DELTAmats) [reply components/avatar-component/AvatarComponent.vue]: naturally after changing the above

File: components/page-component/PageComponent.vue (hunk 1 above)
Comment (DELTAmats) [reply components/page-component/PageComponent.vue]: added text element with name

File: components/page-component/PageComponent.vue (hunk 2 above)
Comment (DELTAmats) [reply components/page-component/PageComponent.vue]: changed

**Pull request merged**